cmd/soroban-cli/src/strval.rs

```diff
@@ -82,14 +82,16 @@ impl Spec {
             | ScType::Symbol
             | ScType::Status
             | ScType::Bytes
-            | ScType::Address
             | ScType::Void
             | ScType::Timepoint
             | ScType::Duration
             | ScType::U256
             | ScType::I256
             | ScType::String
             | ScType::Bool => String::new(),
+            ScType::Address => String::from(
+                "Can be public key (G13..), a contract hash (6c45307) or an identity (alice), ",
```

Comment:
Or an `S...` key?

Reply:
Also, we should get rid of contract hash support here when we do https://github.com/stellar/soroban-tools/issues/403

Reply:
Yeah we could, but it's a bad practice to put a secret in plaintext in a CLI command.   But for scripts I guess it's okay.